build.gradle

```diff
@@ -84,29 +62,32 @@ repositories {
     }
     maven {
         // Jade
-        url "https://www.cursemaven.com"
+        url = "https://www.cursemaven.com"
         content {
             includeGroup "curse.maven"
         }
     }
     flatDir {
-        dirs 'libs'
+        dirs "libs"
     }
 }
 
 minecraft {
     mappings channel: mappingsChannel, version: mappingsVersion
-    accessTransformer = file('src/main/resources/META-INF/accesstransformer.cfg')
+    accessTransformer = file("src/main/resources/META-INF/accesstransformer.cfg")
 
     runs {
-        client {
-            workingDirectory project.file('run')
-            args '-mixin.config=' + mod_id + '.mixins.json'
-            property 'forge.logging.console.level', 'debug'
-            property 'mixin.env.remapRefMap', 'true'
-            property 'mixin.env.refMapRemappingFile', "${projectDir}/build/createSrgToMcp/output.srg"
-            property 'forge.enabledGameTestNamespaces', mod_id
-            jvmArgs('-ea', '-Xmx4G', '-Xms4G')
+        all {
+            args("-mixin.config=${mod_id}.mixins.json")
+
+            property("forge.logging.console.level", "debug")
+            property("forge.enabledGameTestNamespaces", mod_id)
```

Comment:
should this be in all?

Reply:
Yes, to enable /test in run client / server. The difference between test server and gametest server is just that the latter is for CI.